event-bus/src/kafka.rs

```diff
@@ -47,14 +54,15 @@ impl KafkaEventBus {
     }
 
     pub(crate) async fn subscribe(&self, group: &str, topics: &[&str]) -> Result<KafkaConsumer, Error> {
-        let consumer: StreamConsumer = ClientConfig::new()
+        let mut config = self.config.clone();
+        config
             .set("group.id", group)
-            .set("bootstrap.servers", &self.brokers)
             .set("enable.partition.eof", "false")
             .set("session.timeout.ms", "6000")
             .set("enable.auto.commit", "false")
-            .set("auto.offset.reset", "earliest")
-            .create()?;
+            .set("auto.offset.reset", "earliest");
```

Comment:
Here we are setting fixed values and not reading the properties from the user/env, why this behaviour ? Are these default values used as fallback and overrided later ?

Reply:
No these are partly required by the application logic and should not be influenced by the user. Except for the timeout maybe.